corehq/apps/users/views/__init__.py

```diff
@@ -1144,7 +1144,9 @@ def invite_web_user_form(self):
         if invitation:
             assigned_location = invitation.assigned_locations.filter()
             assigned_location_ids = [loc.location_id for loc in assigned_location]
-            primary_location_id = invitation.primary_location.location_id
+            primary_location_id = None
+            if assigned_location_ids:
```

Comment:
Shouldn't this check `primary_location` instead of `assigned_location_ids`?  It should work the same in either case, just seems clearer